include/glow/Base/Type.h

```diff
@@ -595,6 +602,8 @@ struct Type final {
       return sizeof(uint8_t);
     case ElemKind::UInt8FusedFP16QTy:
       return sizeof(uint8_t);
+    case ElemKind::UInt4FusedFP16QTy:
+      return sizeof(uint8_t);
```

Comment:
So far, we don't have "half byte" type in real. For UInt4FusedFP16QTy, I still use "uint8_t" as the base type (1 byte, 2 data). That is why all the places where the "real" type of UInt4FusedFP16QTy is used, I uses "uint8_t".   
I checked the places where "getElementSize()" are used, most of them are memory usage or address  calculation. So if we use e.g. `sizeof(uint8_t)/2`  I think sometime it would have problem. So I think we can still use "sizeof(uint8_t) and based on given LSB/MSB flag to get the proper data.